scripts/ci-e2e.sh

```diff
@@ -48,27 +48,35 @@ export IMAGE_OS=${IMAGE_OS}
 export FORCE_REPO_UPDATE="false"
 export USE_IRSO="${USE_IRSO:-false}"
 EOF
-# if running basic integration or clusterctl-upgrade test skip apply bmhs in dev-env
-if [[ "${GINKGO_FOCUS:-}" == "clusterctl-upgrade" ]] || [[ "${GINKGO_FOCUS:-}" == "basic" ]] || [[ "${GINKGO_FOCUS:-}" == "integration" ]]; then
+
+case "${GINKGO_FOCUS:-}" in
+  'clusterctl-upgrade'|'k8s-upgrade'|'basic'|'integration')
+    # if running basic, integration, k8s upgrade or clusterctl-upgrade test, skip apply bmhs in dev-env
     echo 'export SKIP_APPLY_BMH="true"' >>"${M3_DEV_ENV_PATH}/config_${USER}.sh"
-fi
-if [[ ${GINKGO_FOCUS:-} == "features" ]]; then
+    export GINKGO_SKIP="${GINKGO_SKIP:-} scalability"
+  ;;
+  'features')
     mkdir -p "$CAPI_CONFIG_FOLDER"
```

Comment:
```suggestion
    mkdir -p "${CAPI_CONFIG_FOLDER}"
```
While touching the lines, drive-by fix this.